_posts/static/2017-11-07-self-create-an-user-account.md

```diff
@@ -37,9 +40,14 @@ Content-Type: application/json
 
 Answer if the request succeeds
 
-```
+```json
 200 OK
+{
+  "id": 3,
+  "message": ""
+}
```

Comment:
I don't understand why there is this HTTP response here. Is this a copy / paste error ?

Reply:
I'll check

Reply:
Hi, @btry 

As far as I can tell, this is the response when is 200 ok, the id of the captcha generated

Reply:
OK, I tought this was related to the the authentication step. So, nevermind about my comment.